packages/cli-hermes/src/profileHermes/index.ts

```diff
@@ -0,0 +1,79 @@
+import {logger} from '@react-native-community/cli-tools';
+import {Config} from '@react-native-community/cli-types';
+import {downloadProfile} from './downloadProfile';
+
+type Options = {
+  verbose: boolean;
+  fileName?: string;
+  raw?: boolean;
+  sourcemapPath?: string;
+  generateSourcemap?: boolean;
+};
+
+async function profileHermes(
+  [dstPath]: Array<string>,
+  ctx: Config,
+  options: Options,
+) {
+  try {
+    logger.info(
+      'Downloading a Hermes Sampling Profiler from your Android device...\n',
+    );
+    if (!options.fileName) {
+      logger.info('No filename is provided, pulling latest file\n');
+    }
+    if (options.verbose) {
+      logger.setVerbose(true);
```

Comment:
Done automatically for you - https://github.com/react-native-community/cli/blob/b7778db91e1033e8f5a6338f2e8805c4b6f6d967/packages/cli/src/index.ts#L190